build.sh

```diff
@@ -68,6 +68,11 @@ if image_needs_building "conjur-test:$TAG"; then
   docker build --build-arg "VERSION=$TAG" -t "conjur-test:$TAG" -f Dockerfile.test .
 fi
 
+if image_needs_building "conjur-ubi:$TAG"; then
+  echo "Building image conjur-ubi:$TAG container"
+  docker build --build-arg "VERSION=$TAG" -t "conjur-ubi:$TAG" -f Dockerfile.ubi .
```

Comment:
is this image also going to need the `flatten` command that we use for the Conjur image above?

Reply:
I'm not sure why you are doing this, the image is working perfectly for me without loosing any env variables, worth to ask @cyberark/conjur-core-team as well.

Reply:
based on the [version](https://hub.docker.com/r/guygiat/conjur-ubi-ocp/tags) you pushed to dockerhub (207 MB) vs the size of the `cyberark/conjur` [image on dockerhub](https://hub.docker.com/r/cyberark/conjur/tags) (257 MB), I don't think you need the `flatten` here :)